lib/core/test/integration/Test/Integration/Scenario/CLI/Wallets.hs

```diff
@@ -99,6 +100,27 @@ spec = do
             Stdout outList <- listWalletsViaCLI
             outList `shouldNotContain` walletName
 
+    it "WALLETS_CREATE_06 - Cannot create wallet with wrong port" $ \_ -> do
+        let walletName = "Wallet created via CLI"
+        checkSimpleCreateWallet
+            (Just "1338")
+            "15"
```

Comment:
What is the significance of the number `15`? Is it the minimum mnemonic length? Perhaps we can introduce a named constant `minimumMnemonicLength`? (But then again, perhaps this is already defined elsewhere?)